dev/tracing_tests/test/common.dart

```diff
@@ -52,3 +54,27 @@ Future<void> runFrame(VoidCallback callback) {
   callback();
   return result;
 }
+
+// This binding skips the zones tests. These tests were written before we
```

Comment:
For my own understanding: What do these tests do with zones that required disabling the zone check? From scanning through the tests, it wasn't immediately clear to me.

Reply:
I suspect that `test` introduces a `Zone` to catch (async) errors? But I didn't spend much time trying to track it down, to be honest.